employees/forms.py

```diff
@@ -59,18 +60,31 @@ class ReportForm(forms.ModelForm):
 
     class Meta:
         model = Report
-        fields = ("date", "author", "description", "task_activities", "project", "work_hours")
+        fields = ("date", "author", "description", "project", "task_activities", "work_hours")
         widgets = {"date": DatePickerInput(format="%Y-%m-%d"), "author": HiddenInput}
 
     def __init__(self, *args: Any, **kwargs: Any):
         super(ReportForm, self).__init__(*args, **kwargs)
         author = kwargs["initial"]["author"]
         self.fields["project"].queryset = author.get_project_ordered_by_last_report_creation_date()
+
+        self._filter_task_activities_per_project(author)
+        self._set_last_choices_in_report_form(author)
+
+    def _filter_task_activities_per_project(self, author: CustomUser) -> None:
+        if 'project' in self.data:
+            project_id = int(self.data.get('project'))
```

Comment:
In next commit there is added `try..except` block because if user would put empty string we could get `ValueException`, I made it `pass` because form after this will raise ValidationException and it will be shown to user.